presto-main/etc/catalog/tablestore.properties

```diff
@@ -0,0 +1,12 @@
+#
+# WARNING
+# ^^^^^^^
+# This configuration file is for development only and should NOT be used be
+# used in production. For example configuration, see the Presto documentation.
+#
+
+connector.name=tablestore
+tablestore.endpoint=https://demo.cn-hangzhou.ots.aliyuncs.com
+tablestore.access-key-id=demo
```

Comment:
again, is `demo` intentionally used here? or, we might try other names

Reply:
done